packages/wonder-blocks-form/src/theme/default.ts

```diff
@@ -0,0 +1,28 @@
+import {border} from "@khanacademy/wonder-blocks-tokens";
+
+export default {
+    inputWrapper: {
+        padding: "0px",
+        margin: "0px",
```

Comment:
Could we use `sizing.size_0` for spacing? Do we also want to group this into a `layout` group following the [component token structure](https://khanacademy.atlassian.net/wiki/spaces/WB/pages/3967287319/Component+tokens+structure)?

```
inputWrapper: {
  layout: {
    padding: sizing.size_0,
    margin: sizing.size_0,
  }
}
```